tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -423,6 +424,17 @@ func addSchedulerCommandFunc(cmd *cobra.Command, args []string) {
 func NewScatterRangeSchedulerCommand() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "scatter-range [--format=raw|encode|hex] <start_key> <end_key> <range_name>",
+		Short: "scatter-range will be deprecated in the future, please use scatter-range-scheduler instead",
+		Run:   addSchedulerForScatterRangeCommandFunc,
+	}
+	c.Flags().String("format", "hex", "the key format")
```

Comment:
When will this command be deprecated?
And maybe you can add information in `Deprecated`
ref: https://github.com/spf13/cobra/blob/main/command.go#L102

Reply:
The exact point of deprecation still needs to be discussed with the PM, no conclusion for now.